run.sh

```diff
@@ -122,10 +122,17 @@ elif [ $SYSTEMTESTS_SCENARIO = "LIBRARY_CONF_CUSTOM_HEADERS_LONG" ]; then
     DD_TRACE_HEADER_TAGS=$(docker run system_tests/weblog env | grep DD_TRACE_HEADER_TAGS | cut -d'=' -f2)
     WEBLOG_ENV="DD_TRACE_HEADER_TAGS=$DD_TRACE_HEADER_TAGS,header-tag1:custom.header-tag1,header-tag2:custom.header-tag2"
 
-elif [ $SYSTEMTESTS_SCENARIO = "REMOTE_CONFIG_MOCKED_BACKEND_FEATURES" ]; then
-    export RUNNER_ARGS="scenarios/remote_config/test_remote_configuration.py::Test_RemoteConfigurationUpdateSequenceFeatures"
-    export SYSTEMTESTS_LOG_FOLDER=logs_remote_config_mocked_backend_features
-    export SYSTEMTESTS_LIBRARY_PROXY_STATE='{"mock_remote_config_backend": "FEATURES"}'
+elif [ $SYSTEMTESTS_SCENARIO = "BACKEND_WAF" ]; then
```

Comment:
@cbeauchesne Is this a merge conflict artifact ? should we remove it ?

Reply:
AFAIK this PR did not add this originally, and it should be merge conflict issue.

Reply:
I think so too, let's wait for the others to confirm

Reply:
I must have brought it in accidentally when I tried to get the branch up to date against main. Sorry about that!